doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -1,6 +1,17 @@
 Install OpenCV-Python in Ubuntu {#tutorial_py_setup_in_ubuntu}
 ===============================
 
+@tableofcontents
+
+Table of Contents
+
+1. Installing OpenCV using pip in Ubuntu
+2. Installing OpenCV-Python from Pre-built Binaries
+3. Building OpenCV from source
+4. Optional Dependencies
+5. Downloading OpenCV
+6. Configuring and Installing
```

Comment:
Please try replace to just `[TOC]`. Seems like there is an issue with `@tableofcontents`

Reply:
related: https://github.com/opencv/opencv/issues/24643